scripts/SetupServiceFabric/SetupServiceFabric.sh

```diff
@@ -5,6 +5,11 @@
 # It also sets up Azure Service Fabric CLI
 #
 # Usage: sudo ./SetupServiceFabric.sh
+# Setting up Service Fabric from local service fabric runtime and sdk debain packages is all supported, both two debain packages will be needed for installation.
+# This script should be called with path of sf runtime and sf sdk. Below is the sample
+# sudo ./setup.sh --servicefabricruntime=/mnt/c/Users/sindoria/Downloads/servicefabric_8.2.142.2.deb --servicefabricsdk=/mnt/c/Users/sindoria/Downloads/servicefabric_sdkcommon_1.4.2.deb
```

Comment:
This script should be called with path of sf runtime and sf sdk. Below is the sample
# sudo ./setup.sh --servicefabricruntime="

This script is SetupServiceFabric.sh, why do you say setup.sh ?

Reply:
Fixed typo